### PR TITLE
To review : Dodo Recap

### PR DESCRIPTION
**Pull request by shiftkey**
cc @andrewtobin

### CODE REVIEW

File: _posts/2012-04-16-dodo-recap.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Once boxkite is feature-complete, tested and documented, it will move to a separate repository with NuGet packages available so that people can consume the API. 
+
+There are no immediate plans for boxkite to support other framework versions, as:
+
+ - it uses a number of .NET 4.5-specific features - including async/await and WebAuthenticationBroker.
+ - there are a number of [existing libraries](https://dev.twitter.com/docs/twitter-libraries#dotnet) out there for .NET apps 
+ - I would prefer to focus on the dodo application itself in the short term.
+
+ If you still have the urge to check out a shiny new Twitter API library, @mabster spent the week building a .NET 4 client called [Budgie](http://matthamilton.net/budgie) that uses the Task Parallel Library heavily.
```
Comment (andrewtobin) [inline _posts/2012-04-16-dodo-recap.md]: [@mabster](http://twitter.com/mabster) linkified :P

**Comment by andrewtobin** [comment]
Looks good, and more and more I want to look at getting that UX I've been discussing for years in there - once I can actually log into the thing.

Do we want to discuss the thing?

File: _posts/2012-04-16-dodo-recap.md (hunk 1 above)
Comment (shiftkey) [inline _posts/2012-04-16-dodo-recap.md]: What's a mabster?

Comment (andrewtobin) [inline _posts/2012-04-16-dodo-recap.md]: a bird worshipping code dweller

**Pull request merged**